modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -29,3 +31,43 @@ function oe_content_publication_post_update_00001_update_field_labels(array &$sa
 function oe_content_publication_post_update_00002() {
   \Drupal::service('module_installer')->install(['oe_content_documents_field']);
 }
+
+/**
+ * Create Contact reference field and update form display.
+ */
+function oe_content_publication_post_update_00003() {
+  $module_installer = \Drupal::service('module_installer');
+  if (!\Drupal::service('module_handler')->moduleExists('oe_content_organisation_reference')) {
+    $module_installer->install(['oe_content_organisation_reference']);
+  }
+  $module_installer->install(['path', 'inline_entity_form']);
+
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_publication') . '/config/post_updates/00003_contact_reference');
```

Comment:
```suggestion
  $storage = new FileStorage(drupal_get_path('module', 'oe_content_publication') . '/config/post_updates/00003_add_contact_reference');
```

Reply:
**Also important:** we need to add the config hash here too, as these configuration are provided by `./config/install`, otherwise the labels that are provided by them will not be translatable.